src/bundles/comms.stanza

```diff
@@ -311,7 +311,51 @@ specification is present in NXP's references
 @member ws Word Select - typically for left vs right differentiation
 @member sd Data Signal
 <DOC>
-public pcb-bundle i2s :
+
+public pcb-enum jsl/bundles/comms/I2SPins :
+  I2S-MCK 
+  I2S-SDI
+
+pcb-bundle i2s-b (pins:Collection<I2SPins>):
   pin sck
   pin ws
-  pin sd
\ No newline at end of file
+  pin sdo
```

Comment:
The `SDO` vs `SDI` distinction here is because of the microcontroller. Not because of the protocol. 

Unfortunately - the bundle language in JITX here is insufficient to actually describe the system we need. 

I don't think we should do this. 

I propose that we:
1.  We should instead leave it as a 3-pin bundle per the specification.  
2. We don't add the I2S supports bundle to the STM32 or whatever MCU yet. 

This way the user can create a `I2S` supports statement that meets their needs instead of us trying to force a solution that will never work properly because the language is deficient.